media/css/mozorg/about.scss

```diff
@@ -7,17 +7,17 @@ $image-path: '/media/protocol/img';
 
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 @import '~@mozilla-protocol/core/protocol/css/components/billboard';
-@import '~@mozilla-protocol/core/protocol/css/components/hero';
 
 // * -------------------------------------------------------------------------- */
 // Hero image
 
-.mzp-c-hero.t-mission {
+.mzp-c-call-out.mzp-t-hero {
```

Comment:
**issue (non-blocking)**: text is center-aligned on prod but left aligned locally
<img width="1130" alt="about-local" src="https://user-images.githubusercontent.com/19650432/212360875-a74d8fab-4386-47c0-903b-35068b20a073.png">
vs prod
<img width="1137" alt="about-prod" src="https://user-images.githubusercontent.com/19650432/212360938-f90d884b-989e-4a9f-8546-615792575842.png">

**suggestion**: center-align to match prod